doorstop/cli/test/test_all.py

```diff
@@ -699,9 +699,15 @@ def test_publish_document_without_child_links(self):
 
     def test_publish_document_without_body_levels(self):
         """Verify 'doorstop publish' can create output without body levels."""
-        self.assertIs(None, main(['publish', 'tut', '--no-body-levels']))
```

Comment:
To ensure that no one's existing workflow breaks, `--no-body-levels` should still be support and this should become two tests, one for `--no-levels=body` and another for the legacy method.